### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by gabrielsroka**
i added word "Date" to "Time". because i had opened the page, searched for "Date" but couldn't find anything.

## Description:
- Describe your changes. This will most likely be a copy-paste of your commit
  message(s), which should be descriptive and informative.

### Resolves:
* [Issue #X](#X)

* [OKTA-XXXXXX](https://oktainc.atlassian.net/browse/OKTA-XXXXXX)

### CODE REVIEW

File: _source/_reference/okta_expression_language/index.md [hunk 1]
```diff
@@ -248,13 +248,13 @@ Use this function to retrieve properties about the user identified with the spec
     * Example: `user.getLinkedObject("manager").appuser("Salesforce").lastName
     * Example Result: `Benioff`
 
-### Time Functions
+### Date and Time Functions
```
Comment (jmelberg-okta) [inline _source/_reference/okta_expression_language/index.md]: We should avoid changing headers, as external links that depend on this deep-link will break.

Comment (jmelberg-okta) [reply _source/_reference/okta_expression_language/index.md]: Second, the table below outlines `Time` based functions. We don't reference a `Date` object, which could confuse our readers.

**Comment by bretterer** [comment]
Closing this PR as stale.  If you would like to make the above changes, please feel free to re-open this PR

**Comment by gabrielsroka** [comment]
Can we at least at a sentence of the top of that section that says "below
are _date_ and time functions" ? Many programming languages use the word
"date" or "datetime", not "time"

**Comment by bretterer** [comment]
Always open to having this PR updated and re-opened

**Pull request closed without merging**